### PR TITLE
New object type CMOD - Customer Enhancement Project

### PR DESCRIPTION
**Pull request by mbtools**
Test repo https://github.com/abapGit-tests/CMOD

### CODE REVIEW

**Comment by larshp** [comment]
#151

File: src/objects/zcl_abapgit_object_cmod.clas.abap [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+
+
+  METHOD zif_abapgit_object~exists.
+
+    DATA: lv_name TYPE modact-name.
+
+    SELECT SINGLE name FROM modact INTO lv_name WHERE name = ms_item-obj_name.
+    IF sy-subrc = 0.
+      rv_bool = abap_true.
+    ENDIF.
```
Comment (larshp) [inline src/objects/zcl_abapgit_object_cmod.clas.abap:130]:
```suggestion
    rv_bool = boolc( sy-subrc = 0 ).
```

File: src/objects/zcl_abapgit_object_cmod.clas.abap [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+
+
+
+CLASS ZCL_ABAPGIT_OBJECT_CMOD IMPLEMENTATION.
+
+
+  METHOD zif_abapgit_object~changed_by.
+
+    SELECT SINGLE anam FROM modattr INTO rv_user WHERE name = ms_item-obj_name.
+    IF sy-subrc = 0.
```
Comment (g-back) [inline src/objects/zcl_abapgit_object_cmod.clas.abap:23]:
```suggestion
    IF sy-subrc <> 0.
```

**Comment by larshp** [comment]
thanks 👍

**Pull request merged**